crates/core_arch/src/arm/neon.rs

```diff
@@ -0,0 +1,1187 @@
+#[allow(unused_imports)]
```

Comment:
Remove all the `#[allow(unused_imports)]` and `#[cfg(target_arch = "arm")]` from  this file.